faunadb-java-dsl/src/test/java/com/faunadb/client/dsl/DslSpec.java

```diff
@@ -986,6 +986,30 @@ public void shouldEvalTimeExpression() throws Exception {
     assertThat(res.to(TIME).get(), equalTo(new Instant(0).plus(Duration.standardHours(4))));
   }
 
+  @Test
+  public void shouldEvalToStringExpression() throws Exception {
+    Value res = query(ToString(Value(100))).get();
+    assertThat(res.to(STRING).get(), equalTo("100"));
+  }
+
+  @Test
+  public void shouldEvalToNumberExpression() throws Exception {
+    Value res = query(ToNumber(Value("100"))).get();
```

Comment:
passing `100.0` to that function, will it return a double?

Reply:
It will.

Reply:
Do you want a test for that? Semantics are tested in the database.

Reply:
Ideally yes, if possible with a different float point number as well

Reply:
These aren't fixed precision numbers (i.e. decimals), so that might not work as easily as you might think. I'll give it a shot though.